README.md

```diff
@@ -11,36 +11,41 @@ Using fluent-plugin-logdna, you can send the logs you collect with Fluentd to Lo
 * Add the contents below to `/etc/fluent/fluent.conf`. For td-agent, use `/etc/td-agent/td-agent.conf`:
 * Alternative install without td-agent is: `gem install fluent-plugin-logdna`
 
+## Configuration
+
+### Configuration Parameters
+- `api_key`: Ingestion Key, *Required*
+- `hostname`: Hostname, *Required*
+- `app`: App Name, *Optional*
+- `mac`: Mac Address, *Optional*
+- `ip`: IP Address, *Optional*
+- `tags`: Comma-separated List of Tags, *Optional*
+- `request_timeout`: Request Timeout for Streaming Logs, *Optional*
+  - **Note**: it supports `s` by default and `ms` suffixes
```

Comment:
IMO, none of these make sense without types to go with them.  If they're all strings, we should say that and be consistent of this through the whole doc.

Reply:
We need to be consistent about our doc formatting.  A lot of time was put into this, and it's this is the style used by the nodejs core docs. https://www.npmjs.com/package/@logdna/logger

Reply:
Well, it's different - we could make it consistent across all code libraries like in `@logdna/logger` but in the plugin it's all about using the key value in the configuration file which doesn't allow to specify the type - I have updated the labels for each field but I know we need to update `request_timeout` handling since it is kinda weird - instead of `30 s` or `30000 ms`, it should be just `30000` representing `ms` - that replacement is gonna cause major version update but we can add new `timeout` field and slowly deprecate `request_timeout` field.